test/unit/org/apache/cassandra/tools/nodetool/InvalidateCIDRPermissionsCacheTest.java

```diff
@@ -142,11 +142,15 @@ public void testInvalidateAllCidrPermissions()
         assertThat(roleA.hasAccessFromIp(ipAddr)).isTrue();
         assertThat(roleB.hasAccessFromIp(ipAddr)).isTrue();
         assertThat(originalReadsCount).isEqualTo(getCidrPermissionsReadCount());
+        assertThat(((CassandraCIDRAuthorizer) DatabaseDescriptor.getCIDRAuthorizer()).has(ROLE_A)).isTrue();
```

Comment:
As mentioned above, this unit test just invokes AuthCache's invalidate() function which re-inits the cache. So we don't need has(role) function.